datumaro/plugins/delete_transforms.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import annotations
```

Comment:
- Please put this code to `transforms.py`
- Please use `black` for autoformatting. We don't use `autopep8`